server/models/notification.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Notification = sequelize.define('Notification', {
+    notification: {
+      allowNull: false,
+      type: DataTypes.STRING,
+    },
+  }, {});
+  Notification.associate = (models) => {
+    // associations can be defined here
+    Notification.belongsTo(models.User, {
+      foreignKey: 'userId',
+    });
+  };
+  return Notification;
+};
```

Comment:
Is hound no longer in the repo?

Reply:
It is. I am not sure as to whether it has stopped working. Would look into this.

Reply:
It is actually working. Hound just picked up some violation.

Reply:
You still haven't fixed the end of line issue.